docs/pages/web/popovereducational.js

```diff
@@ -0,0 +1,628 @@
+    setOpenA(true);
+    setOpenB(true);
+  }, []);
+
+  return (
+    <React.Fragment>
+      <Flex direction="column" width="100%" height="100%">
+        <Flex.Item flex="grow">
+          <IconButton
+            accessibilityLabel="This IconButton represents a new feature"
+            iconColor="darkGray"
+            icon="pin"
+            onClick={() => {}}
+            ref={anchorRefA}
+            size="lg"
+        />
+        </Flex.Item>
+        <Flex.Item flex="grow">
+          <IconButton
+            accessibilityLabel="This IconButton represents a new feature"
+            iconColor="darkGray"
+            icon="pin"
+            onClick={() => {}}
+            ref={anchorRefB}
+            size="lg"
+          />
+        </Flex.Item>
+
+        </Flex>
+        {openA && (
+          <PopoverEducational
+            size="flexible"
+            anchor={anchorRefA.current}
+            idealDirection="right"
+            onDismiss={() => {}}
+            message="Ads allow you to reach more people who matter to you. When audiences engage with your ad, you'll be able to track performance here."
+          />
+        )}
+        {openB && (
+          <PopoverEducational
+            anchor={anchorRefB.current}
+            idealDirection="right"
+            onDismiss={() => {}}
+            message="Ads allow you to reach more people who matter to you. When audiences engage with your ad, you'll be able to track performance here."
+
+          />
+        )}
+    </React.Fragment>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Visibility on page load"
+          description={`
+PopoverEducational's positioning algorithm requires that the anchor element renders before PopoverEducational is rendered. If PopoverEducational should be visible on page load, use \`useEffect\` to toggle the visibility after the first render.
+`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+
+  React.useEffect(() => {
+    setOpen(true);
+  }, []);
+
+  return (
+    <Flex alignItems="center" justifyContent="center" height="100%" width="100%">
+      <IconButton
+        accessibilityLabel="This IconButton represents a new feature"
+        iconColor="darkGray"
+        icon="pin"
+        onClick={() => {}}
+        ref={anchorRef}
+        size="lg"
+      />
+      {open && (
+        <PopoverEducational
+          anchor={anchorRef.current}
+          idealDirection="right"
+          onDismiss={() => {}}
+          message="This PopoverEducational is visible on initial page load"
+        />
+      )}
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <QualityChecklist component={generatedDocGen?.displayName} />
+
+      <MainSection name="Related">
```

Comment:
Seems like Popover should be here as well?

Reply:
POpover is goig to become a building block, so I'd keep it separated from here.